planning/planning_debug_tools/scripts/perception_replayer/perception_reproducer.py

```diff
@@ -66,6 +72,7 @@ def __init__(self, args):
             prev_stamp = stamp
 
         average_ego_odom_interval = sum(time_diffs) / len(time_diffs)
+        average_ego_odom_interval *= 1.2  # slow down the publication speed.
```

Comment:
Can the factor be a parameter ? (e.g., `publishing_speed_factor` with a default value of `1.2`)

Reply:
Great idea